test-suite/start-test-environment.sh

```diff
@@ -40,11 +40,26 @@ docker-compose -f docker-compose.yml up -d activemq redis dynamodb ps_db outboun
 docker-compose -f docker-compose.yml up db_migration
 docker-compose -f docker-compose.yml rm -f db_migration
 
-cd docker/snomed-database-loader
-./load_release-postgresql.sh ${SNOMED_FILE_LOCATION}
-cd ../..
-cd docker/snomed-immunization-loader
-./load_immunization_codes.sh
-cd ../..
+snomedexists=$(docker-compose exec -u postgres ps_db psql -d patient_switching -AXqtc "SELECT EXISTS (SELECT 1 FROM information_schema.schemata WHERE schema_name='snomedct')")
```

Comment:
it looks like "SELECT COUNT(*) FROM information_schema.schemata WHERE schema_name='snomedct'" can be used here as well which would make the script more unified and better maintainable... unless there is a specific reason why "SELECT EXISTS..." should be used... I tested with "SELECT COUNT..." and it works OK...